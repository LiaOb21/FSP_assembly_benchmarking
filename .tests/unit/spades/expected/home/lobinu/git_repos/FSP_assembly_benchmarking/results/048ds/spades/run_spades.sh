set -e
true
true
/home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/bin/spades-core /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K21/configs/config.info /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K21/configs/careful_mode.info
/home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/bin/spades-core /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K33/configs/config.info /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K33/configs/careful_mode.info
/home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/bin/spades-core /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/configs/config.info /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/configs/careful_mode.info
/home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/bin/python3 /home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/share/spades/spades_pipeline/scripts/copy_files.py /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/before_rr.fasta /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/before_rr.fasta /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/assembly_graph_after_simplification.gfa /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/assembly_graph_after_simplification.gfa /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/final_contigs.fasta /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/contigs.fasta /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/first_pe_contigs.fasta /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/first_pe_contigs.fasta /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/strain_graph.gfa /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/strain_graph.gfa /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/scaffolds.fasta /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.fasta /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/scaffolds.paths /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.paths /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/assembly_graph_with_scaffolds.gfa /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/assembly_graph_with_scaffolds.gfa /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/assembly_graph.fastg /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/assembly_graph.fastg /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/final_contigs.paths /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/contigs.paths
true
true
/home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/bin/python3 /home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/share/spades/spades_pipeline/scripts/correction_iteration_script.py --corrected /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/contigs.fasta --assembled /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/misc/assembled_contigs.fasta --assembly_type contigs --output_dir /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades --bin_home /home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/bin
/home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/bin/python3 /home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/share/spades/spades_pipeline/scripts/correction_iteration_script.py --corrected /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.fasta --assembled /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/misc/assembled_scaffolds.fasta --assembly_type scaffolds --output_dir /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades --bin_home /home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/bin
true
/home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/bin/python3 /home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/share/spades/spades_pipeline/scripts/breaking_scaffolds_script.py --result_scaffolds_filename /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.fasta --misc_dir /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/misc --threshold_for_breaking_scaffolds 3
true
