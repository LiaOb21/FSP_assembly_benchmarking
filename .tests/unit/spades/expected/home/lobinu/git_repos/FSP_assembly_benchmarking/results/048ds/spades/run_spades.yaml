- STAGE: Before start
  args: []
  config_dir: ''
  del_after: []
  job_uuid: SPAdes_Before_start_5Y8RJUDTN6GTVJH6K50GLU0OC9EONKO6
  mpi_support: false
  output_files: []
  path: 'true'
  short_name: before_start
- STAGE: Assembling
  args: []
  config_dir: ''
  del_after: []
  job_uuid: SPAdes_Assembling_RBEZ5Q8KFW2ZY05JZM682LYLBB01POT3
  mpi_support: false
  output_files: []
  path: 'true'
  short_name: as_start
- STAGE: K21
  args:
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K21/configs/config.info
  config_dir: K21
  del_after: []
  job_uuid: hpcSPAdes_K21_S8XH0BA0453HM3SYQNRINQSNOR949KP3
  mpi_support: true
  output_files: []
  path: /home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/bin/{spades_core}
  short_name: k21
- STAGE: K33
  args:
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K33/configs/config.info
  config_dir: K33
  del_after: []
  job_uuid: hpcSPAdes_K33_WC7XCQ0KHJM0JX7WJZNGOGOPN9PBCH19
  mpi_support: true
  output_files: []
  path: /home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/bin/{spades_core}
  short_name: k33
- STAGE: K55
  args:
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/configs/config.info
  config_dir: K55
  del_after: []
  job_uuid: hpcSPAdes_K55_TPDML2UJCVTYDOVAC3HYAW7L5TTQBD7I
  mpi_support: true
  output_files: []
  path: /home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/bin/{spades_core}
  short_name: k55
- STAGE: Copy files
  args:
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/share/spades/spades_pipeline/scripts/copy_files.py
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/before_rr.fasta
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/before_rr.fasta
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/assembly_graph_after_simplification.gfa
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/assembly_graph_after_simplification.gfa
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/final_contigs.fasta
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/contigs.fasta
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/first_pe_contigs.fasta
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/first_pe_contigs.fasta
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/strain_graph.gfa
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/strain_graph.gfa
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/scaffolds.fasta
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.fasta
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/scaffolds.paths
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.paths
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/assembly_graph_with_scaffolds.gfa
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/assembly_graph_with_scaffolds.gfa
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/assembly_graph.fastg
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/assembly_graph.fastg
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/final_contigs.paths
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/contigs.paths
  config_dir: ''
  del_after:
  - .bin_reads
  - tmp/spades_65e8e8vc
  job_uuid: SPAdes_Copy_files_JYHJ6PAO5W07SHCBLC1TNHDAJODY5XW6
  mpi_support: false
  output_files: []
  path: /home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/bin/python3
  short_name: copy_files
- STAGE: Assembling
  args: []
  config_dir: ''
  del_after: []
  job_uuid: SPAdes_Assembling_W9I19D4LTSR6GMBRG33CASOIA7Y2FGHU
  mpi_support: false
  output_files: []
  path: 'true'
  short_name: as_finish
- STAGE: Breaking scaffolds
  args:
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/share/spades/spades_pipeline/scripts/breaking_scaffolds_script.py
  - --result_scaffolds_filename
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.fasta
  - --misc_dir
  - /home/lobinu/git_repos/FSP_assembly_benchmarking/results/048ds/spades/misc
  - --threshold_for_breaking_scaffolds
  - '3'
  config_dir: ''
  del_after: []
  job_uuid: SPAdes_Breaking_scaffolds_BYWDTIPTQGBVX5A3ZTJII2YAYS01AAB5
  mpi_support: false
  output_files: []
  path: /home/lobinu/git_repos/FSP_assembly_benchmarking/.snakemake/conda/c9d8da4aeba45c11e035c12a2dc44715_/bin/python3
  short_name: bs
- STAGE: Terminate
  args: []
  config_dir: ''
  del_after:
  - configs
  job_uuid: SPAdes_Terminate_NFD3FVBQ96RO7QG4XHP2282X6LPHANGK
  mpi_support: false
  output_files: []
  path: 'true'
  short_name: terminate
